tests/coprocessor/test_select.rs

```diff
@@ -391,13 +395,13 @@ impl<'a> Select<'a> {
         self
     }
 
-    fn build(self) -> Request {
+    fn build(self) -> (Request, Request) {
         self.build_with(&[0])
     }
 
-    fn build_with(mut self, flags: &[u64]) -> Request {
+    fn build_with(mut self, flags: &[u64]) -> (Request, Request) {
```

Comment:
Add a comment for this function. `(Request, Request)` is not a returned value which is easy to understand.